suite-native/module-staking-management/src/components/StakingBalancesOverviewCard.tsx

```diff
@@ -0,0 +1,135 @@
+import {
+    NativeStakingRootState,
+    selectAPYByAccountKey,
+    selectRewardsBalanceByAccountKey,
+    selectStakedBalanceByAccountKey,
+} from '@suite-native/staking';
+
+const stakingItemStyle = prepareNativeStyle(utils => ({
+    flexDirection: 'row',
+    alignItems: 'center',
+    gap: utils.spacings.extraSmall,
+    paddingBottom: utils.spacings.small,
+}));
+
+const stakingCardStyle = prepareNativeStyle(utils => ({
+    marginTop: utils.spacings.medium,
+}));
+
+const stakingWrapperStyle = prepareNativeStyle(utils => ({
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'flex-start',
+    paddingBottom: utils.spacings.medium,
+}));
+
+const separatorStyle = prepareNativeStyle(utils => ({
+    borderBottomWidth: 1,
+    borderBottomColor: utils.colors.borderElevation1,
+}));
+
+type StakingBalancesCardProps = {
+    accountKey: string;
+    handleToggleBottomSheet: (value: boolean) => void;
+};
+
+export const StakingBalancesOverviewCard = ({
+    accountKey,
+    handleToggleBottomSheet,
+}: StakingBalancesCardProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const networkSymbol = useSelector((state: AccountsRootState) =>
+        selectAccountNetworkSymbol(state, accountKey),
+    );
+
+    const apy = useSelector((state: NativeStakingRootState) =>
+        selectAPYByAccountKey(state, accountKey),
+    );
+
+    const stakedBalance = useSelector((state: NativeStakingRootState) =>
+        selectStakedBalanceByAccountKey(state, accountKey),
+    );
+    const rewardsBalance = useSelector((state: NativeStakingRootState) =>
+        selectRewardsBalanceByAccountKey(state, accountKey),
+    );
+
+    if (!networkSymbol) return null;
+
+    return (
+        <TouchableOpacity onPress={() => handleToggleBottomSheet(true)}>
+            <Card style={applyStyle(stakingCardStyle)}>
+                <Box style={applyStyle(stakingWrapperStyle)}>
+                    <Box flex={1}>
+                        <Box style={applyStyle(stakingItemStyle)}>
+                            <Icon name="lock" color="textSubdued" size="medium" />
+                            <Text color="textSubdued" variant="label">
+                                <Translation id="staking.staked" />
+                            </Text>
+                        </Box>
+                        <CryptoAmountFormatter
+                            value={stakedBalance}
+                            network={networkSymbol}
+                            decimals={5}
+                            color="textDefault"
+                            variant="titleSmall"
+                        />
+                        <Box flexDirection="row">
+                            <Text color="textSubdued">≈</Text>
+                            <CryptoToFiatAmountFormatter
+                                value={stakedBalance}
+                                network={networkSymbol}
+                                color="textSubdued"
```

Comment:
```suggestion
                                color="textSubdued"
                                isBalance
```

Otherwise it does not show correct fiat amount

Reply:
good catch done